packages/mdc-dom/README.md

```diff
@@ -34,6 +34,7 @@ Function Signature | Description
 --- | ---
 `closest(element: Element, selector: string) => ?Element` | Returns the ancestor of the given element matching the given selector (which may be the element itself if it matches), or `null` if no matching ancestor is found.
 `matches(element: Element, selector: string) => boolean` | Returns true if the given element matches the given CSS selector.
+`estimateScrollWidth(element: Element) => number`  | 
```

Comment:
Do we need a short description for this?

Reply:
Done